www/go/modules/community/addressbook/install/install.sql

```diff
@@ -107,6 +107,13 @@ CREATE TABLE `addressbook_user_settings` (
   `defaultAddressBookId` int(11) DEFAULT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=COMPACT;
 
+CREATE TABLE `addressbook_filter_contact_map`(
```

Comment:
So I see you moved this to it's own module. We can remove this then right?

Reply:
We have moved this code from addressbook to filteraddressbook module. Please refer to the recent commit